src/components/form-field/FormField.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import {Text, TextInput, TextInputProps, View} from 'react-native';
+
+import {formFieldStyles} from './styles';
+
+interface FormFieldProps {
+  errorText: string;
+  fieldLabel: string;
+}
+
+export const FormField = ({
+  value,
+  onChangeText,
+  errorText,
+  fieldLabel,
+  ...props
+}: FormFieldProps & TextInputProps) => {
```

Comment:
```suggestion
interface FormFieldProps extends TextInputProps {
  errorText: string;
  fieldLabel: string;
}

export const FormField = ({
  value,
  onChangeText,
  errorText,
  fieldLabel,
  ...props
}: FormFieldProps) => {
```

Detalhezinho, mas a ideia aqui é que a nossa convenção de nomenclatura das props é que a interface do componente X, chamada de XProps tenha na própria interface todos os campos.